ui/perfherder/helpers.js

```diff
@@ -332,3 +341,201 @@ export const getGraphsLink = function getGraphsLink(seriesList, resultSets, time
 
   return `perf.html#/graphs${createQueryParams(params)}`;
 };
+
+export const httpParamSerializer = paramsObject => map(paramsObject, (value, key) => `${key}=${value}`).join('&');
```

Comment:
You can replace uses of this with the helper `createQueryParams` in the `helpers/url.js` file.